civihr_default_theme/scss/components/_header.scss

```diff
@@ -1,3 +1,5 @@
+$chr-menu-responsive-breakpoint: 1250px;
```

Comment:
This should go in the base/_variables.scss. Also it seems that the prefix we are using here is for the variables is `crm_` (this i think is my fault, back in the day when i just joined), so let's try to keep things consistent